sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts

```diff
@@ -240,30 +240,34 @@ describe("Certificates client - create, read, update and delete", () => {
     const base64PKCS12 = certificateSecret.value!;
     fs.writeFileSync("pkcs12.p12", Buffer.from(base64PKCS12, "base64"));
 
-    // Obtaining only the public certificate.
-    // We send "-passin 'pass:'" because our self-signed certificate doesn't specify a password on its issuer.
-    childProcess.execSync(
-      "openssl pkcs12 -in pkcs12.p12 -out pkcs12.crt.pem -clcerts -nokeys -passin pass:"
-    );
+    // skip the following in playback mode because the certificate placeholder value
+    // isn't in valid pkcs12 format.
+    if (!isPlaybackMode()) {
```

Comment:
@maorleger @sadasant do you think it is acceptable to skip this verification for playback?

Reply:
could we generate a valid pkcs12 certificate placeholder? 🤔

Reply:
I thought about that but don't think it brings much value as it will always pass the verification.

Reply:
I’ll leave it up to @maorleger

Reply:
I am fine with this because we run it although we now have 3 `skip` here:

Line 223 we call `recorder.skip`
Then in line 225 we skip if not node
Finally we skip if not playback here.

I thought that `recorder.skip` would skip playback and record mode (but not in live mode) which is what we want here - is that not already happening?

Reply:
Oh sorry, I missed `browser` - let's just change 

```ts
recorder.skip("browser", "This test uses the file system.");
```

To

```ts
recorder.skip(undefined, "<reasons>");
```

Reply:
@maorleger I could skip the whole test but you don't want some coverage with playback tests for node?

Reply:
Personally I would prefer a clear and obvious `skip` point as opposed to having it spread across the test. In addition, there are no assertions in the lines above so other than exceptions in playback mode we won't get any meaningful coverage.

Between using openssl, a SecretClient, and fs, this feels about as integration-y as they come so I am fine only running it in live mode